.travis.yml

```diff
@@ -40,15 +40,15 @@ jobs:
         - sudo mount --make-rshared /
         - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_VERSION}/bin/linux/amd64/kubectl
         - sudo chmod +x kubectl && sudo mv kubectl /usr/local/bin/
-        - curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 
```

Comment:
This change does not belong to this PR description

Reply:
that just came along with formatter, I can fix this as well.